prime-router/settings/organizations.yml

```diff
@@ -2522,9 +2522,10 @@
         initialTime: 00:00
         timeZone: EASTERN
       transport:
-        type: SFTP
-        host: sftp
-        port: 22
-        filePath: ./upload
-        credentialName: DEFAULT-SFTP
+        type: FTPS
```

Comment:
Why this change? Why not continue to use the credential from the credStore?

Reply:
Ah, I meant to just duplicate this entry. Will address